packages/devtools_app/lib/src/screens/network/network_service.dart

```diff
@@ -13,38 +13,55 @@ class NetworkService {
 
   final NetworkController networkController;
 
-  /// Updates the last refresh time to the current time.
+  /// Updates the last Socket data refresh time to the current time.
   ///
   /// If [alreadyRecordingHttp] is true it's unclear when the last refresh time
   /// would have occurred, so the refresh time is not updated. Otherwise,
-  /// [NetworkController.lastRefreshMicros] is updated to the current
+  /// [NetworkController.lastSocketDataRefreshMicros] is updated to the current
   /// timeline timestamp.
   ///
-  /// Returns the current timestamp.
-  Future<int> updateLastRefreshTime({bool alreadyRecordingHttp = false}) async {
+  /// Returns the current timeline timestamp.
+  Future<int> updateLastSocketDataRefreshTime({
+    bool alreadyRecordingHttp = false,
```

Comment:
this seems strange now that we need to know the state of Http recording in order to update the socket data refresh time. How are HTTP and socket recording related?

Reply:
I've now changed it to check whether socket data is already being recorded instead.